test_autolens/analysis/test_result.py

```diff
@@ -92,9 +101,12 @@ def test__source_plane_light_profile_centre(analysis_imaging_7x7):
         redshift=1.0, light=al.lp.SersicSph(centre=(5.0, 6.0), intensity=2.0)
     )
 
-    tracer = al.Tracer.from_galaxies(galaxies=[lens, source_0, source_1])
+    tracer = al.Tracer(galaxies=[lens, source_0, source_1])
+
+    instance = af.ModelInstance()
+    instance = tracer

```

Comment:
is this correct? 

it seems strange because you are overriding instance with traces before using instance.

and also perhaps you can pick a better name instead of instance :-)

Reply:
Thanks for the suggestion :).

This change ended up getting reverted cause it didnt do the wonders to the API that I was hoping, but I agree it would of needed a bit of refactoring / variable renaming.